src/components/VDisk/VDisk.tsx

```diff
@@ -62,23 +76,26 @@ export const VDisk = ({data = {}, nodes, compact}: VDiskProps) => {
 
     return (
         <React.Fragment>
-            <VDiskPopup data={data} nodes={nodes} anchorRef={anchor} open={isPopupVisible} />
-            <div className={b()} ref={anchor} onMouseEnter={showPopup} onMouseLeave={hidePopup}>
-                {vDiskPath ? (
-                    <InternalLink to={vDiskPath} className={b('content')}>
-                        <DiskStateProgressBar
-                            diskAllocatedPercent={data.AllocatedPercent}
-                            severity={data.Severity}
-                            compact={compact}
-                        />
-                    </InternalLink>
-                ) : (
+            <VDiskPopup
```

Comment:
as i may recall component rendering order its better to place popup after main div so when it is passed to popup anchor is defined on the very first render